README.md

```diff
@@ -104,13 +108,11 @@ so that your commits provide a good history of the changes you are making.  To b
 
 ### Add Unit Tests for New Code
 
-If you're adding a new feature to the project, please make sure to include adequate [xUnit](http://xunit.github.io/)
-tests with your change.  In this project, we have chosen write out unit tests in a way that uses the
-actual PowerShell environment rather than extensive interface mocking.  This allows us to be sure that
-our features will work in practice.  
+If you're adding a new feature to the project, please make sure to include adequate [nUnit](http://nunit.org/)
```

Comment:
did this change from xUnit to nUnit for the whole repo?

Reply:
When I wrote this code however many years ago I was annoyed with the crappy xunit asserts and was trying to push people to nunit.
I don't know if it took root when I dropped this PR.

Reply:
So, most of the services are using nUnit now, and just a few that are still using xUnit framework.  Going by numbers, it is about 2% of test cases using xUnit framework and rest is on nUnit.